superduper/components/vector_index.py

```diff
@@ -221,20 +221,33 @@ def trigger_ids(self, query: Query, primary_ids: t.Sequence):
         )
         return ids
 
+    def _create_predict_job(self, db, callable, deps, ids, job_id=None):
```

Comment:
Why predicting on `VectorIndex`? This was handled by `Listener` before.

Reply:
the name is wrong, I will rename it to `create_vector_sync_job`

Reply:
Please do this prior to merge

Reply:
Please do this prior to merge

Reply:
done